pkg/apis/build/v1alpha2/build_pod_test.go

```diff
@@ -903,34 +762,11 @@ func testBuildPod(t *testing.T, when spec.G, it spec.S) {
 			}
 			assert.Equal(t, append(append([]string{
 				"-layers=/layers",
-				"-analyzed=/layers/analyzed.toml",
-				"-run-image=builderregistry.io/run"},
```

Comment:
I think this is correct that the run-image wouldn't be there anymore, because we've upgraded the platform API version,  and _think_ that really is the main change,  but uh... cc @natalieparellano  :)

Reply:
We should still provide `-run-image` to the analyzer. It is only removed as a flag to the exporter on newer platforms.